libs/ui/src/components/NeoModal/NeoModalBody.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="modal-width">
+    <header
+      class="px-6 is-flex is-justify-content-space-between border-bottom is-align-items-center"
+      :class="{
+        'py-5': !loading,
+        'py-4': loading,
+      }">
+      <NeoSkeleton
+        v-if="loading"
+        rounded
+        width="150"
+        height="35"
+        no-margin
+        border-radius="4rem"
```

Comment:
is border-radius prop really needed?

Reply:
this skeleton has rounder borders.

Reply:
@exezbcz let's keep/unify the same border radius on all skeleton